bin/image-list.rb

```diff
@@ -17,6 +17,14 @@
   end
 end
 
+File.open(File.join(helm, "values.yaml")).each do |line|
+  ["DOWNLOADER","EXECUTOR","UPLOADER"].each do |image|
+    /^\s+EIRINI_#{image}_IMAGE: ["'].+\/([^\/]*)["']$/.match(line) do |match|
+      images[match[1]] = true
+    end
+  end
+end
+
```

Comment:
This whole block should not be necessary because the case is already handled by the "config variables" section above because the image name now no longer includes a slash.

Reply:
I could be wrong but I think this script is run on very first un-altered staging values.yaml. That `values.yaml` file should have whatever we define [in the role-manifest](https://github.com/SUSE/scf/pull/2906/files#diff-a65976ef35c13467fe140d46c6ca0d3aR3786) which is the cap-staging url with the slashes in it.

Reply:
Ah, yes, in that case just loop over the `values` hash that has already been setup. Something like this (untested braindump):

```
values['env'].each_value do |value|
    /^registry\.suse\.com\/cap-staging\/(.*)$/.match(value) { |match| images[match[1]] = true }
end
```

I don't know if we can guarantee that this will be run with Ruby 2.5+, then we could make it more readable like

```
values['env'].each_value do |value|
    image = value.delete_prefix("registry.suse.com/cap-staging/")
    images[image] = true unless image.nil?
end
```

Reply:
I'll go with the first suggestion to be safe. Regular expressions are very readable (said no one ever).